src/azure-cli/azure/cli/command_modules/vm/linter_exclusions.yml

```diff
@@ -136,11 +152,51 @@ vm install-patches:
         package_name_masks_to_include:
             rule_exclusions:
             - option_length_too_long
+vm deallocate:
+    parameters:
+        hibernate:
+            rule_exclusions:
+            - missing_parameter_help
+vm delete:
+    parameters:
+        force_deletion:
+            rule_exclusions:
+            - missing_parameter_help
 image create:
     parameters:
         os_type:
             rule_exclusions:
             - missing_parameter_help
+image show:
+    parameters:
+        expand:
+            rule_exclusions:
+            - missing_parameter_help
+ppg update:
+    parameters:
+        include_colocation_status:
+            rule_exclusions:
+            - missing_parameter_help
+restore-point collection wait:
+    parameters:
+        expand:
+            rule_exclusions:
+            - missing_parameter_help
+restore-point wait:
+    parameters:
+        expand:
+            rule_exclusions:
+            - missing_parameter_help
+sig image-version wait:
+    parameters:
+        expand:
+            rule_exclusions:
+            - missing_parameter_help
```

Comment:
These are all existing parameters, why these linter issues haven't they been scanned and detected before?

Reply:
Additionally, I can see the help message for these parameters through `-h`

Reply:
Hi @msyyc, we noticed that the comment of some params are changed from ":param" to ":keyword" (e.g., https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-compute_29.1.0/sdk/compute/azure-mgmt-compute/azure/mgmt/compute/v2022_11_01/operations/_virtual_machines_operations.py#L1662 and https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-compute_30.0.0/sdk/compute/azure-mgmt-compute/azure/mgmt/compute/v2022_11_01/operations/_operations.py#L15734).
Since our CLI would read the SDK function params and apply the comments of SDK as the help message for some params, this change would bring a breaking change to our CLI.
May I ask why do you make this change? Is it possible to keep it consistent with the previous one?

Reply:
![image](https://github.com/Azure/azure-cli/assets/70930885/d7d0fb70-98d2-449a-814d-ab2b100b6ac3)
For latest compute, all query and header parameter will be `keyword-only` instead of `positoinal`. So its doc type changes from `param` to `keyword`

Reply:
> These are all existing parameters, why these linter issues haven't they been scanned and detected before?

Since the doc type of these params are changed from `param` to `keyword` in Python SDK side and our CLI read comments from SDK as arguments help messages, the help messages are lost when using the latest SDK package. We would add them manually to avoid breaking changes.